multicluster/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
It would have been good to use the same boilerplate as Antrea to avoid discrepancies and duplication. https://github.com/antrea-io/antrea/blob/main/hack/boilerplate/license_header.go.txt. I can probably take care of this myself later.

Reply:
thanks,